app/assets/javascripts/bastion/components/views/bst-edit.html

```diff
@@ -4,7 +4,10 @@
       <i class="fa fa-edit inline-icon"></i>
       <i class="fa fa-remove inline-icon" ng-click="delete($event)" ng-show="deletable"></i>
     </span>
-    <span class="editable-value">{{ displayValue }}</span>
+    <span>
+      <i class="{{ iconClass }}" ng-show="iconShow"></i>
+      <span class="editable-value">{{ displayValue }}</span>
```

Comment:
I left the editable-value class present to keep the tests happy

Reply:
Does it break tests in katello too or just bastion?  I'd argue that we should remove this since you removed the CSS below.

Reply:
Just bastion because it's using that css class to find the element. I'm totally OK with removing. but do you have any advice on a sound way to modify the finder if I were to remove the editable-value class from it?

https://github.com/Katello/bastion/blob/6e1ddbe30365963335e4675c61be765e6376652b/test/components/bst-edit.directive.test.js#L70

Reply:
Oh I see.  Yeah, referencing it in any other way would look rather hacky so I'm find leaving this in.

(Big fan of it showing the line of code inline by the way, thanks github!)